includes/collection/class-following.php

```diff
@@ -54,14 +53,14 @@ class Following {
 	/**
 	 * Follow a user.
 	 *
-	 * Please do not use this method directly, use `Activitypub\follow` instead.
+	 * Please do not use this method directly, use `\Activitypub\follow` instead.
 	 *
-	 * @see Activitypub\follow
+	 * @see \Activitypub\follow
 	 *
 	 * @param \WP_Post|int $post    The ID of the remote Actor.
 	 * @param int          $user_id The ID of the WordPress User.
 	 *
-	 * @return \WP_Post|\WP_Error The ID of the Actor or a WP_Error.
+	 * @return int|false|\WP_Post|\WP_Error The Outbox ID or false on failure, the Actor post or a WP_Error.
```

Comment:
@pfefferle The possible return values of this method are a bit confusing. It's a WP_Error with an invalid actor, the outbox item ID on success and false on failure, and if we didn't get to adding it to the outbox, the actor post. Would it be possible to limit it to two types?

Reply:
should we do this as part of this PR or in a follow up?

can you file an issue if it should be a follow up?